nextest-runner/src/platform.rs

```diff
@@ -21,22 +24,55 @@ pub struct BuildPlatforms {
     /// In the future, this will become a list of target triples once multiple `--target` arguments
     /// are supported.
     pub target: Option<TargetTriple>,
```

Comment:
@sunshowers what's the plan for multiple targets here? Should we change this to a `Vec` here or should we just make `RustBuildMetaSummary::target_platforms_v2` a `Vec`?

Reply:
I'd just make `target_platforms_v2` a `Vec`. Actually supporting multiple targets is a lot more work, but let's make sure the relatively unchanging backend structures support this.

Reply:
Done.